src/stats/actions.ts

```diff
@@ -152,60 +142,87 @@ function handleActionCompute(state: PlayerActionState, indices: PlayerIndexedTyp
+  incrementCount("attackCount.jab2", currentAnimation === State.ATTACK_JAB2);
+  incrementCount("attackCount.jab3", currentAnimation === State.ATTACK_JAB3);
+  incrementCount("attackCount.jabm", currentAnimation === State.ATTACK_JABM);
+  incrementCount("attackCount.dash", currentAnimation === State.ATTACK_DASH);
+  incrementCount("attackCount.ftilt", isForwardTilt(currentAnimation));
+  incrementCount("attackCount.utilt", currentAnimation === State.ATTACK_UTILT);
+  incrementCount("attackCount.dtilt", currentAnimation === State.ATTACK_DTILT);
+  incrementCount("attackCount.fsmash", isForwardSmash(currentAnimation));
+  incrementCount("attackCount.usmash", currentAnimation === State.ATTACK_USMASH);
+  incrementCount("attackCount.dsmash", currentAnimation === State.ATTACK_DSMASH);
+  incrementCount("attackCount.nair", currentAnimation === State.AERIAL_NAIR);
+  incrementCount("attackCount.fair", currentAnimation === State.AERIAL_FAIR);
+  incrementCount("attackCount.bair", currentAnimation === State.AERIAL_BAIR);
+  incrementCount("attackCount.uair", currentAnimation === State.AERIAL_UAIR);
+  incrementCount("attackCount.dair", currentAnimation === State.AERIAL_DAIR);
+
+  // GnW is weird and has unique IDs for some moves
+  if (playerFrame.internalCharacterId === 0x18) {
+    incrementCount("attackCount.jab1", currentAnimation === State.GNW_JAB1);
+    incrementCount("attackCount.jabm", currentAnimation === State.GNW_JABM);
+    incrementCount("attackCount.dtilt", currentAnimation === State.GNW_DTILT);
+    incrementCount("attackCount.fsmash", currentAnimation === State.GNW_FSMASH);
+    incrementCount("attackCount.nair", currentAnimation === State.GNW_NAIR);
+    incrementCount("attackCount.bair", currentAnimation === State.GNW_BAIR);
+    incrementCount("attackCount.uair", currentAnimation === State.GNW_UAIR);
+  }
 
-    incrementCount("groundTechCount.in", currentAnimation === State.FORWARD_TECH && facingOpponent);
-    incrementCount("groundTechCount.in", currentAnimation === State.BACKWARD_TECH && !facingOpponent);
-    incrementCount("groundTechCount.neutral", currentAnimation === State.NEUTRAL_TECH);
-    incrementCount("groundTechCount.away", currentAnimation === State.BACKWARD_TECH && facingOpponent);
-    incrementCount("groundTechCount.away", currentAnimation === State.FORWARD_TECH && !facingOpponent);
+  // Throws
+  incrementCount("throwCount.up", currentAnimation === State.THROW_UP);
+  incrementCount("throwCount.forward", currentAnimation === State.THROW_FORWARD);
+  incrementCount("throwCount.down", currentAnimation === State.THROW_DOWN);
+  incrementCount("throwCount.back", currentAnimation === State.THROW_BACK);
 
-    incrementCount("wallTechCount.success", currentAnimation === State.WALL_TECH);
-    incrementCount("wallTechCount.fail", currentAnimation === State.MISSED_WALL_TECH);
+  // Techs
+  let opponentDir = 1;
+  if (playerFrame.positionX! > opponentFrame.positionX!) {
+    opponentDir = -1;
```

Comment:
```suggestion
  const opponentDir = (playerFrame.positionX! > opponentFrame.positionX!)  ? -1 : 1;
```
nit: can we use a ternary here?